test/controllers/admin/comfy/v2/dashboard_controller_test.rb

```diff
@@ -60,38 +60,93 @@ class Comfy::Admin::V2::DashboardControllerTest < ActionDispatch::IntegrationTes
     assert_equal Date.today.beginning_of_month, assigns(:start_date)
     assert_equal Date.today.end_of_month, assigns(:end_date)
 
-    assert_equal [page_visit_event_1_page_2.id], assigns(:page_visit_events).pluck(:id)
-    assert_equal [click_event_1_page_2.id], assigns(:click_events).pluck(:id)
-    assert_equal [video_watch_event_1_page_2.id], assigns(:video_view_events).pluck(:id)
-
-    assert_equal [page_visit_event_1.id], assigns(:previous_period_page_visit_events).pluck(:id)
-    assert_equal [click_event_1.id], assigns(:previous_period_click_events).pluck(:id)
-    assert_equal [video_watch_event_1.id], assigns(:previous_period_video_view_events).pluck(:id)
+    # Page Visit Events
+    page_visit_data = assigns(:page_visit_events)
+    assert_equal page_visit_data[:events_count], 1
+    assert_equal page_visit_data[:previous_period_events_count], 1
+    assert_equal page_visit_data[:pie_chart_data]['Single time visitor'], 1
```

Comment:
@alis-khadka is there a reason to switch to hard coded right hand values?

Reply:
Nope, I will update these.